1-exercises/C-array-every/exercise.js

```diff
@@ -8,9 +8,9 @@ let group = ["Austine", "Dany", "Swathi", "Daniel"];
 let groupIsOnlyStudents; // complete this statement
 
 if (groupIsOnlyStudents) {
```

Comment:
Please check this to find more about syntax and how to use this method: 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every
https://www.w3schools.com/jsref/jsref_every.asp
https://www.youtube.com/watch?v=Arbk085MWjc